### ISSUE TITLE
RNFS.stat reports file not exists, but RNFS.exists reports file exists.

### ISSUE THREAD

**Issue by williams-voon** [itinance/react-native-fs#525]
The code that call RNFS.stat (version 2.9.11) works.

But after I upgrade RNFS to the latest version, and RNFS.stat always run into exception, says file does not exists.

My sample code:

```
RNFS.stat(path).then(
(stat)=>{}
).catch((err) => {
        console.log(err.message, err.code); //run into exception
      });
```

The file do exists.

**Comment by JQiMeng** [comment]
I meet this problem too,please help .how to fix it

**Comment by haotangio** [comment]
I'm having this problem too. Can someone help?

**Comment by vysotsky** [comment]
Yes, also having this issue. However it cannot be reproduced on 2.9. 
Seems like https://github.com/itinance/react-native-fs/pull/480 has broken something.

**Comment by yuvaraj119** [comment]
Any fixes or update on this issue.

**Comment by canyara** [comment]
same on 2.10.14, any solution?

**Comment by encubos** [comment]
Same issue here, I can't get the real path from a URI
Running Android
react-native-fs -- version: "2.16.6"

Error: **File does not exist**

my URI es like:
content://com.android.providers.downloads.documents/document/15

**Comment by arthedza** [comment]
Are any updates on this?

**Comment by matteodanelli** [comment]
Same error of @encubos. Any solution or workaround to resolve real path from URI?

**Comment by afilp** [comment]
Is this bug fixed? I have the same problem, thanks!

**Comment by flyskywhy** [comment]
`rn-fetch-blob` works for me, ref to https://github.com/itinance/react-native-fs/issues/756#issuecomment-554937928
